packages/wxt/src/core/builders/vite/plugins/devHtmlPrerender.ts

```diff
@@ -113,29 +113,30 @@ export function devHtmlPrerender(
     {
       name: 'wxt:virtualize-inline-scripts',
       apply: 'serve',
-      resolveId(id) {
-        // Resolve inline scripts
-        if (id.startsWith(virtualInlineScript)) {
-          return '\0' + id;
-        }
-
-        // Ignore chunks during HTML file pre-rendering
-        if (id.startsWith('/chunks/')) {
-          return '\0noop';
-        }
+      resolveId: {
+        filter: {
+          id: [new RegExp(`^${virtualInlineScript}`), /^\/chunks\//],
```

Comment:
You can probably remove `/^\/chunks\//` as you only care about returning `'\0' + id` if the filter matches

Reply:
That's also true, good job on your review :)